### ISSUE TITLE
On first install/run of hydrolinks, error on set_local_path

### ISSUE THREAD

**Issue by lawinslow** [lawinslow/hydrolinks#32]
````
> hydrolinks::set_local_path('d:/hydrolinks')
Error in file(file, ifelse(append, "a", "w")) : 
  cannot open the connection
In addition: Warning message:
In file(file, ifelse(append, "a", "w")) :
  cannot open file 'C:\Users\LAWINS~1\AppData\Local\hydrolinks\hydrolinks/path': No such file or directory
````

I think it is from trying to save the path file without having first created the AppData directory.

**Comment by lawinslow** [comment]
```
Browse[2]> file.exists(rappdirs::user_data_dir(appname = "hydrolinks"))
[1] FALSE
```

**Comment by lilyhahn** [comment]
fixed in #28.
